Docs/README-csharp-v2.md

```diff
@@ -0,0 +1,255 @@
+# Usage examples - C&#35;
+
+## Basics
+
+The PhaxioClient class is the entry point for any Phaxio operation (you'll also need to add ```using Phaxio.V2```).
+
+    var phaxio = new PhaxioV2Client(key, secret);
```

Comment:
It's namespaced under V2, so new users would put "using Phaxio.V3" and existing users could continue to use "using Phaxio.V2".